src/main/java/OUTPUT.txt

```diff
@@ -0,0 +1,5 @@
+[T] [X] cock   
```

Comment:
Really love the test cases mentioned here, just personal preference, if the task type and Marked is placed side by side it would be nicer